### ISSUE TITLE
MissingPluginException(No implementation found for method startUi on channel flutter_auth_ui)

### ISSUE THREAD

**Issue by rubenaalders** [koji-1009/flutter_auth_ui#20]
I love this package, but I keep hitting the follow error on iOS
`Unhandled Exception: MissingPluginException(No implementation found for method startUi on channel flutter_auth_ui)
#0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:157:7)
<asynchronous suspension>
#1      FlutterAuthUi.startUi (package:flutter_auth_ui/flutter_auth_ui.dart:112:18)
<asynchronous suspension>
#2      UserSingleton.login (package:k_pop_fans_app/util/util_user.dart:37:20)
<asynchronous suspension>`

It works perfectly on Android but somehow I can't get it to work on iOS. Not sure what I could have missed. Also took an extensive look at your example and they seem to be alike.

I use all of these from Firebase

> flutter_auth_ui: ^1.0.0
> firebase_core: ^0.5.3
> firebase_analytics: ^5.0.2
> firebase_database: ^4.4.0
> firebase_auth: 0.18.4+1
> firebase_crashlytics: ^0.2.4
> firebase_performance: ^0.4.3
> firebase_messaging: ^7.0.3

**Comment by koji-1009** [comment]
Hi, @rubenaalders 
Thank you for using.

I'm very sorry, but I'm currently having problems building with the latest Firebase libraries.
For this reason, the new version of the latest library(using firebase_core 0.7.0) has not been released.

Perhaps the version specification by `^` has a bad effect, so could you try to modify the version specification as below?
`flutter_auth_ui: 1.0.0` uses `Firebase/CoreOnly v6.33.0`. [link](https://github.com/koji-1009/flutter_auth_ui/blob/1.0.0/example/ios/Podfile.lock#L20)
(Don't forget to run `flutter clean` and update Podfile.lock 😄 )

```txt
flutter_auth_ui: 1.0.0
firebase_core: 0.5.3
firebase_analytics: 6.3.0
firebase_database: 4.4.0
firebase_auth: 0.18.4+1
firebase_crashlytics: 0.2.4
firebase_performance: 0.4.3
firebase_messaging: 7.0.3
```

**Comment by rubenaalders** [comment]
Hi @koji-1009,

Thanks for the quick response. I have tried your suggestion and even tried `flutter pub cache repair`  and removing both the `pubspec.lock` and `podfile.lock` but to no avail.

I have tried it exactly as suggested
```
flutter_auth_ui: 1.0.0
firebase_core: 0.5.3
firebase_analytics: 6.3.0
firebase_database: 4.4.0
firebase_auth: 0.18.4+1
firebase_crashlytics: 0.2.4
firebase_performance: 0.4.3
firebase_messaging: 7.0.3
cloud_firestore: 0.14.4
```

I have no clue how to further debug or get to the bottom of this error..

**Comment by koji-1009** [comment]
I'm very very sorry!
I found a coding mistake, so I'll fix it and release the latest version of firebase.

**Comment by koji-1009** [comment]
I published v1.1.0.
Probably, fixed in this version.
<https://pub.dev/packages/flutter_auth_ui/versions/1.1.0>

**Comment by rubenaalders** [comment]
Hi,

I'd like to confirm that updating to version 1.1.0 does indeed fix the above issue!

Thank you very much for the effort you put into this package. It's much appreciated.
